### ISSUE TITLE
luca wants to know why blank perturbations invert boundaries between x and y

### ISSUE THREAD

**Issue by gszep** [gszep/reaction-diffusion#6]
An example show below, where the diffusivities of X,Y and B are equal, and the boundary between X (blue) and Y(yellow) is perturbed with a circle where B (black) is set to 0.9 and X = Y = 0. Why do we see the boundary inverting?

![screenshot from 2018-01-22 17-32-07](https://user-images.githubusercontent.com/17977117/35235660-ad6bd5ea-ff9c-11e7-8572-212d41ac8cbb.png) ![screenshot from 2018-01-22 17-32-11](https://user-images.githubusercontent.com/17977117/35235656-ad08331e-ff9c-11e7-9b8a-687903084df9.png) ![screenshot from 2018-01-22 17-32-16](https://user-images.githubusercontent.com/17977117/35235659-ad3ad558-ff9c-11e7-8089-5cb02a7c61ea.png)

* is it a bug? Are the values of X and Y really zero within the perturbation? 
* Is this a genuine memory effect?

**Comment by gszep** [comment]
This was indeed **bug** due to the clipping of values between 0 and 1 in 8-bit levels. We must allow any real value with 32-bit precision. Unfortunately the `three.js` limited me to the usage of 8-bit textures. With commit 3ac4d51ac9bbc8b7a953c4e6ffb2a927b8b788de I factored out the library and I coded a custom javascript framework in WebGL2.0. This upgrade also allows me to use a more recent versions of OpenGL Shaders, with which I can generate Monte Carlo quality random numbers.
